tests/sentry/hybrid_cloud/test_user_option.py

```diff
@@ -0,0 +1,116 @@
+import time
+
+import pytest
+
+from sentry.services.hybrid_cloud.user_option import user_option_service
+from sentry.testutils.factories import Factories
+from sentry.testutils.hybrid_cloud import use_real_service
+from sentry.testutils.silo import all_silo_test
+
+
+@pytest.mark.django_db(transaction=True)
+@all_silo_test
+@use_real_service(user_option_service, None)
+def test_user_option_service():
+    objects = [1, dict(a=dict(b=3)), "asdf", 9873, [1, 2, 3], 511]
+
+    u1 = Factories.create_user()
+    u2 = Factories.create_user()
+
+    # snowflake id generation issues...
```

Comment:
@RyanSkonnord This is a good example of where the snowflake id generation logic breaks down.  Remove the `time.sleep` and actually these tests fail due to inability to generate Snowflake ids.  There are a few such bugs.  I think we can simplify the snowflake implementation and prevent this sort of issue.